player/public/coreplugins/org.ekstep.videorenderer-1.1/renderer/plugin.js

```diff
@@ -0,0 +1,324 @@
+/**
+ * @description Launcher to render the Video or youtube URL's
+ * @extends {class} org.ekstep.contentrenderer.baseLauncher
+ * @author Gourav More <gouav_m@tekditechnologies.com>
+ */
+
+org.ekstep.contentrenderer.baseLauncher.extend({
+    _time: undefined,
+    supportedStreamingMimeType: "application/x-mpegURL",
+    isStreaming: EkstepRendererAPI.getGlobalConfig().streaming || false,
+    messages: {
+        noInternetConnection: "Internet not available. Please connect and try again.",
+        unsupportedVideo: "Video URL not accessible"
+    },
+    currentTime: 1,
+    videoPlayer: undefined,
+    stageId: undefined,
+    heartBeatData:{},
+    enableHeartBeatEvent:false,
+    _constants: {
+        mimeType: ["video/mp4", "video/x-youtube", "video/webm"],	
+        events: {	
+            launchEvent: "renderer:launch:video"	
```

Comment:
why single event is defined here? what about other events ?`"renderer:overlay:mute"`